cli/state.rs

```diff
@@ -314,10 +319,13 @@ impl ThreadSafeState {
     Ok(ThreadSafeState(Arc::new(state)))
   }
 
-  pub fn add_child_worker(&self, worker: WebWorker) -> u32 {
+  pub fn add_child_worker(&self, _worker: &WebWorker) -> u32 {
     let worker_id = self.next_worker_id.fetch_add(1, Ordering::Relaxed) as u32;
-    let mut workers_tl = self.workers.lock().unwrap();
-    workers_tl.insert(worker_id, worker);
+    println!("WARNING broken add_child_worker {}", worker_id);
+    // TODO(ry) Fix this... we can't have owned references to workers. so what
+    // goes in the table?
+    // let workers_tl = self.workers.lock().unwrap();
+    // workers_tl.insert(worker_id, worker);
```

Comment:
A few months ago we had something called "shared future" that was returned from worker and it was stored in the table - I don't think that's a good idea though, host worker needs to have "full" control over child worker (like termination, emptying pending futures, etc)